src/Entities/Payout.php

```diff
@@ -0,0 +1,586 @@
+<?php
+/**
+ * This source file is proprietary and part of Rebilly.
+ *
+ * (c) Rebilly SRL
+ *     Rebilly Ltd.
+ *     Rebilly Inc.
+ *
+ * @see https://www.rebilly.com
+ */
+
+namespace Rebilly\Entities;
+
+use Rebilly\Rest\Entity;
+
+/**
+ * Class Payout.
+ */
+final class Payout extends Entity
+{
+    /**
+     * @return string
+     */
+    public function getAmount()
+    {
+        return $this->getAttribute('amount');
+    }
+
+    /**
+     * @param $value
+     *
+     * @return $this
+     */
+    public function setAmount($value)
+    {
+        return $this->setAttribute('amount', $value);
+    }
+
+    /**
+     * @return string
+     */
+    public function getCurrency()
+    {
+        return $this->getAttribute('currency');
+    }
+
+    /**
+     * @param $value
+     *
+     * @return $this
+     */
+    public function setCurrency($value)
+    {
+        return $this->setAttribute('currency', $value);
+    }
+
+    /**
+     * @return string
+     */
+    public function getCreatedTime()
+    {
+        return $this->getAttribute('createdTime');
+    }
+
+    /**
+     * @return string
+     */
+    public function getProcessedTime()
+    {
+        return $this->getAttribute('processedTime');
+    }
+
+    /**
+     * Can be specified only if transaction was processed outside (isProcessedOutside = true).
+     *
+     * @param $value
+     *
+     * @return $this
+     */
+    public function setProcessedTime($value)
+    {
+        return $this->setAttribute('processedTime', $value);
+    }
+
+    /**
+     * @return string
+     */
+    public function getScheduledTime()
+    {
+        return $this->getAttribute('scheduledTime');
```

Comment:
I think there is no such property in Payout definition.

Reply:
This one right? https://github.com/Rebilly/api-definitions/pull/190/files#diff-5f86adb90ebb814d81b2bea1f57c4b0dR83